### DISCUSSION TITLE
WAV Playback Quiet and Noisy

### DISCUSSION THREAD

**Discussion by musiq021** [pschatzmann/arduino-audio-tools#981] [Q&A] [answered]
Recorded a WAV file using streams-audiokit-sd_wav, ( had to change to RXTX mode otherwise file was silence as noted in recent post ).

But when playing back that WAV file with player-sd-audiokit ( changing the Helix MP3 decoder to WAVDecoder ), the audio is very quiet with a lot of white noise on top. 

Any ideas?

( FWIW, the audio plays back fine in VLC, and audio sounds fine in the mic to RAM demo. )

**Comment by pschatzmann** [comment, marked as answer]
Fee free to play with the volume and the mic gain.

**Reply by musiq021** [reply]
The recording / mic gain appears fine, it has good volume in playback on the computer. 

But with the following code, with volume set to 90%, playback is barely audible : https://pastebin.com/8gTSnwhQ

**Comment by pschatzmann** [comment]
Since the volume is handled via the AudioKit hardware it does not make any sense to restrict the volume via the Player as well!
Make sure that either the Player or the AudioKit is set to max output volume!
